src/app/components/bitcoin-transaction-item/bitcoin-transaction-icon.tsx

```diff
@@ -1,32 +1,42 @@
-import { Box, BoxProps, Circle, Flex, color } from '@stacks/ui';
+import { Circle, CircleProps, Flex } from 'leather-styles/jsx';
 
 import { BitcoinTx } from '@shared/models/transactions/bitcoin-transaction.model';
 
-import { BtcIcon } from '@app/components/icons/btc-icon';
+import { getColorFromBitcoinTx, isBitcoinTxInbound } from '@app/common/transactions/bitcoin/utils';
+import { BtcIcon } from '@app/ui/components/icons/btc-icon';
 
-import { IconForTx, colorFromTx } from './utils';
+import { ArrowDownIcon } from '../icons/arrow-down-icon';
+import { ArrowUpIcon } from '../icons/arrow-up-icon';
 
-interface TransactionIconProps extends BoxProps {
+export function TxStatusIcon(props: { address: string; tx: BitcoinTx }) {
+  const { address, tx } = props;
+  if (isBitcoinTxInbound(address, tx)) return <ArrowDownIcon size="13px" />;
+  return <ArrowUpIcon size="13px" />;
+}
+
+interface TransactionIconProps extends CircleProps {
```

Comment:
Our code confuses me sometimes as here we are saying that:
- `interface TransactionIconProps extends CircleProps` 
- Then we give those props to `BitcoinTransactionIcon`
- But when we call the component
    - we only do it once
    - we only pass `transaction` and `address` - `<BitcoinTransactionIcon transaction={transaction} btcAddress={bitcoinAddress} />` 

So it seems there is no need to extend `CircleProps` or to spread any extra props here

Reply:
Yeah it makes sense not to change too much. I just thought this was weird when refactoring. Often I was making changes to appease `extends Props` that weren't really used then